mypy/types.py

```diff
@@ -867,6 +867,53 @@ def deserialize(cls, data: JsonDict) -> 'AnyType':
                        AnyType.deserialize(source) if source is not None else None,
                        data['missing_import_name'])
 
+    def describe(self) -> str:
+        if not mypy.options._based:
+            return 'Any'
+
+        def type_of_any(toa: int = self.type_of_any) -> Optional[str]:
+            if toa == TypeOfAny.unannotated:
+                return "unannotated"
+            elif toa == TypeOfAny.explicit:
+                return None
+            elif toa == TypeOfAny.from_unimported_type:
+                return "from unimported type"
+            elif toa == TypeOfAny.from_omitted_generics:
+                return "from omitted generics"
+            elif toa == TypeOfAny.from_error:
+                return "from error"
+            elif toa == TypeOfAny.special_form:
+                return "special form"
```

Comment:
imo special form should be the one that returns `None`, since "special form" makes no sense to show to the user

Reply:
changed it to `(unknown)`